feature/home/src/commonMain/kotlin/com/tunjid/heron/home/HomeViewModel.kt

```diff
@@ -70,6 +72,7 @@ class ActualHomeViewModel(
     @Suppress("UNUSED_PARAMETER")
     @Assisted
     route: Route,
+    savedStateDataSource: SavedStateDataSource,
```

Comment:
Lets keep the assisted injected fields as the last 2 arguments. While not a requirement for metro (the current dependency injection library used), it is a requirement for others.

Let the constructor be:

```kotlin
class ActualHomeViewModel(
    authRepository: AuthRepository,
    timelineRepository: TimelineRepository,
    savedStateDataSource: SavedStateDataSource,
    writeQueue: WriteQueue,
    navActions: (NavigationMutation) -> Unit,
    @Assisted
    scope: CoroutineScope,
    @Suppress("UNUSED_PARAMETER")
    @Assisted
    route: Route,
)
```

Reply:
Resolved.